packages/shared/styles/components/atoms/SfChevron.scss

```
@@ -2,32 +2,64 @@
 
 .sf-chevron {
   position: relative;
-  width: var(--chevron-size, 1.25rem);
-  height: var(--chevron-size, 1.25rem);
+  width: var(--chevron-width, var(--chevron-size, 1.25rem));
+  height: var(--chevron-height, var(--chevron-size, 1.25rem));
   cursor: pointer;
   &__bar {
     position: absolute;
     top: 50%;
-    background: transparent;
-    transition: all 500ms cubic-bezier(0.25, 1.7, 0.35, 0.8);
+    background: var(--chevron-background, transparent);
+    transition: all 300ms cubic-bezier(0.25, 1.7, 0.35, 0.8);
     &:after {
       content: "";
       display: block;
-      width: calc(var(--chevron-size, 1.25rem) / 2);
-      height: calc(var(--chevron-size, 1.25rem) / 10);
-      background: var(--chevron-background, var(--c-black));
+      width: var(--chevron-size, calc(var(--chevron-width, 1.25rem) / 2));
+      height: var(--chevron-size, calc(var(--chevron-height, 1.25rem) / 10));
+      background: var(--chevron-color, var(--c-black));
     }
     &--left {
       left: calc(var(--chevron-size, 1.25rem) / 10);
-      transform: translateY(-50%) rotate(var(--chevron-rotate, 45deg));
+      transform: translate(
```

```suggestion
      transform: translate3d(
```
try use always transform3d  because for example safari use for this transform hardware-acceleration